public/calls/static/calls-api-2024-05-21.yaml

```
@@ -324,6 +359,75 @@ paths:
                 requiresImmediateRenegotiation: false
                 tracks:
                   - mid: "7"
+  /apps/{appId}/sessions/{sessionId}/tracks/update:
+    put:
+      tags:
+        - Change tracks
+      summary: Change tracks by reusing existing transceivers
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/ChangeTracksRequest"
+            examples:
+              reuse_transceiver:
+                description: Reuse an existing transceiver for a new track
+                value:
+                  tracks:
+                    some-track-name:
+                      location: "remote"
+                      sessionId: "2a45361d5fd7cc14eface0587c276c94"
+                      trackName: "other-track-name"
+                      mid: "7"
+              reuse_with_simulcast:
+                description: Reuse an existing transceiver with simulcast preferences
+                value:
+                  tracks:
+                    trackID2:
+                      location: "remote"
+                      sessionId: "2a45361d5fd7cc14eface0587c276c94"
+                      trackName: "simulcast-track"
+                      mid: "8"
+                      simulcast:
+                        preferredRid: "h"
+                        fallbackStrategy: "leastBandwidth"
+      security:
+        - secret: []
+      parameters:
+        - in: path
+          name: appId
+          schema:
+            type: string
+          required: true
+          description: WebRTC application ID
+        - in: path
+          name: sessionId
+          schema:
+            type: string
+          required: true
+          description: Current PeerConnection session ID
+      responses:
+        "200":
+          description: OK
+          headers:
+            vary:
+              schema:
+                type: string
+                example: Origin
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/UpdateTracksResponse"
+              examples:
+                success:
+                  value:
+                    requiresImmediateRenegotiation: false
+                    tracks:
+                      trackID1:
+                        mid: "7"
+                        sessionId: "2a45361d5fd7cc14eface0587c276c94"
+                        trackName: "new-track-name"
```

Does the `new-track-name` only apply to the use case of replacing a track?
For the simulcast scenario the track stays the same. So I assumed we would not return a `trackName` in that case.

---

I think we should try to return what we have here. I don't see why returning is a bad thing?

---

I think `"new-track-name"` is confusing. It's fine to keep `trackName` though IMO.
```suggestion
                        trackName: "track-name"
```